docs/conf.py

```diff
@@ -98,7 +98,7 @@
             "name": "SynopticPy",
             "url": "https://blaylockbk.github.io/SynopticPy/_build/html/",
         },
-        {"name": "Herbie", "url": "https://blaylockbk.github.io/Herbie/_build/html/"},
+        {"name": "Herbie", "url": "https://herbie.readthedocs.io/"},
```

Comment:
You would like to use a different project name here, right?

Reply:
Oh nevermind. You are actually linking to Herbie here.

Reply:
Yep! It's super easy to apply this to my other repos now that I have a template to follow thanks to you. It'll take a few days to find all the links between my projects, but I'm working on it 😁